packages/react-search-ui-views/src/__tests__/layouts/__snapshots__/LayoutSidebar.test.js.snap

```diff
@@ -0,0 +1,27 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`renders correctly 1`] = `
+<Fragment>
+  <button
+    className="sui-layout-sidebar-toggle"
+    hidden={true}
+    onClick={[Function]}
+    type="button"
+  >
+    Show Filters
+  </button>
+  <div
+    className="sui-layout-sidebar"
+  >
+    <button
+      className="sui-layout-sidebar-toggle"
+      hidden={true}
+      onClick={[Function]}
+      type="button"
+    >
+      Done Filtering
```

Comment:
This is nit-picky, but "Save Filters" might be wording as it promotes an actual action that the user takes to modify the action of the computer/phone ("Save") versus a state-of-being that the user achieves.

Reply:
Also, I just posted this on the `.test` file.... and I also realize it's Monday.
![giphy](https://user-images.githubusercontent.com/2472911/61227570-2aac3e00-a6ea-11e9-8351-82d038949006.gif)

Reply:
I'm good with that change! Just to check, does "Save Filters" also make sense for when you're clearing/have zero filters checked? Or should we eventually make a separate "Clear Filters" button for that anyway?

Reply:
For now, I think it's fine. I do agree that a "clear filters" button would be great in the future. And when the user clicks "clear filters," it will automatically clear _and_ collapse the filters.